lib/spack/spack/cmd/make_installer.py

```diff
@@ -31,33 +34,33 @@ def make_installer(parser, args):
     """
     if(sys.platform == 'win32'):
         output_dir = args.output_dir
-
+        cmake_spec = Spec('cmake')
+        cmake_spec.concretize()
```

Comment:
This strikes me as a little simplistic, since it's possible an installed version of cmake might not match the default spec?

Reply:
Thanks for the suggestion. For now, let's merge into the windows-support branch to allow for testing. We can re-visit before this goes into develop.